specification/storage/Microsoft.BlobStorage/models.tsp

```diff
@@ -934,10 +934,14 @@ model BlobFlatListSegment {
 /** Represents a page list. */
 model PageList {
   /** The page ranges. */
-  @Xml.name("PageRange") pageRange?: PageRange[];
+  @Xml.unwrapped
+  @Xml.name("PageRange")
+  pageRange?: PageRange[];
 
   /** The clear ranges. */
-  @Xml.name("ClearRange") clearRange?: ClearRange[];
+  @Xml.unwrapped
+  @Xml.name("ClearRange")
+  clearRange?: ClearRange[];
```

Comment:
Just confirming, are these properties really unwrapped in the xml response? Have you tested with an actual service response?